conf/index.js

```diff
@@ -12,7 +12,8 @@ const defaultConf = {
   ...require('./config-bots'),
   ...require('./config-contracts'),
   ...require('./config-notification'),
-  ...require('./config-repos')
```

Comment:
I don't understand why is highlighting this change. It was done in the branch you are trying to merge.
There's more cases like this, not sure why it happens.

Reply:
It's highlighted because we added a ',' and a further require() on line 16